src/Harpoon.cpp

```diff
@@ -49,6 +49,16 @@ void Harpoon::RequestChunks() {
    }
 }
 
+
+Harpoon::Battling Harpoon::Cancel() {
+   FreeChunk();
+   RequestChunks();
+   zstr_sendf (mDealer, "<CANCEL>");
```

Comment:
I've done that on the battling side.

I guess I can call EnumToString(::Withdraw) instead of hard coding in the string